discord_slash/context.py

```diff
@@ -321,7 +321,12 @@ def __init__(
             )
             self.component = self.origin_message.get_component(self.custom_id)
 
-        self.selected_options = _json["data"]["values"] if self.component_type == 3 else None
+        self.selected_options = None
+
+        if self.component_type == 3:
+            self.selected_options = (
+                _json["data"]["values"] if "values" in _json["data"].keys() else []
+            )
```

Comment:
```suggestion
            self.selected_options = _json["data"].get("values", [])
```
This can be simplified.